### DISCUSSION TITLE
Having a problem with d3d11 what could be the problem

### DISCUSSION THREAD

**Discussion by tiben20** [haasn/libplacebo#271] [Q&A] [answered]
I tryed a lot to find what is the problem but i tought maybe one of you can tell me righta way what is my problem
Im using a nv12 pictures uploading each plane individually and then use pl_render_image. The output as always teh wrong color. My output is the same color as my backbuffer. What could be the problem the pl_color_repr, pl_color_space , the component mapping or something else?
I posted with the question a good and bad image from what i get

![good color](https://github.com/haasn/libplacebo/assets/70393/b838166b-0306-4f1c-8833-d47f0eb192a8)
![wrong one](https://github.com/haasn/libplacebo/assets/70393/57a4b819-2f06-4c21-ae36-9f1139cb4832)

**Comment by tiben20** [comment]
After testing even if i convert the nv12 to R8G8B8A8 before passing it to pl_render_image i get the same color in results.

**Comment by haasn** [comment, marked as answer]
Looks like wrong `pl_color_system`. Image decodes perfectly if tagged as `PL_COLOR_SYSTEM_BT_709` instead of `PL_COLOR_SYSTEM_RGB`.

**Reply by tiben20** [reply]
> Looks like wrong `pl_color_system`. Image decodes perfectly if tagged as `PL_COLOR_SYSTEM_BT_709` instead of `PL_COLOR_SYSTEM_RGB`.

Thanks i figured out that i was putting the color_system on the frameout
